snake/main.py

```diff
@@ -30,6 +32,7 @@ class Screen:
     WIDTH, HEIGHT = block_size * NUM_BLOCKS_X, block_size * NUM_BLOCKS_Y
     RECT = Rect(0, 0, WIDTH, HEIGHT)
     FPS = 60
+    
```

Comment:
Adding a line here is also against PEP8, as you can see [here](https://www.python.org/dev/peps/pep-0008/#blank-lines).